src/main/java/org/apache/brooklyn/cloudfoundry/entity/VanillaCloudFoundryApplication.java

```diff
@@ -77,7 +79,7 @@
     ConfigKey<Integer> REQUIRED_MEMORY = ConfigKeys.newIntegerConfigKey(
             "cloudfoundry.profile.memory", "Memory allocated for the application (MB)", 512);
 
-    @SetFromFlag("disk")
+    @SetFromFlag("disk_quota")
```

Comment:
I was tried to maintain the manifest namespace, https://docs.cloudfoundry.org/devguide/deploy-apps/manifest.html#disk-quota
But `camelCase` case looks better here, right?